packages/react-docsite-components/src/components/Markdown/Markdown.tsx

```diff
@@ -80,7 +80,7 @@ function getMarkdownProps(subComponentStyles: IMarkdownSubComponentStyles, props
         },
         a: {
           component: MarkdownLink,
-          props: { className: 'ms-mdLink', styles: subComponentStyles.link },
+          props: { className: 'ms-mdLink', styles: subComponentStyles.link, underline: false },
```

Comment:
I'm confused, how did this fix it? I would have expected it to make the links *never* have an underline?

Reply:
Hmm I don't think this is right, because it removes the underlines by default. I think links in markdown really ought to be underlined by default, for example these ones. 
![image](https://user-images.githubusercontent.com/5864305/106056779-e736dc00-60a3-11eb-948c-818ad0dff21d.png)

Reply:
To clarify, the reason I think markdown links ought to be underlined by default is because markdown is for body text, and links in body text should be underlined.

Reply:
Ok thinking about this more, the best solution I can see is different handling of related links all-up (pass an array not use markdown...there's no reason to define a bunch of identically formatted ULs in markdown).

So my suggestion for this PR would be to revert this line (so that all the links *except* related are correctly underlined), and I'll make a follow-up changing the related link formatting.